main/bind/APKBUILD

```diff
@@ -68,6 +74,13 @@ prepare() {
 	default_prepare
 	cd "$builddir"
 
+	# Fetch and verify named.root
+	gpg --import < "$srcdir/verisign-grs-nstld-key.asc"
+	rm -rf named.root named.root.sig
+	curl -RLO 'https://www.internic.net/domain/named.root{.sig,}'
```

Comment:
I don't think the `{.sig,}` part works in busybox ash. Its a bash extension.

I don't think we should fetch this from `prepare()`, as it does not make the build reproducible. We may not end up with same .apk if we rebuild this in the future.

Reply:
@ncopa curl is handling the brace expansion in the URL itself, not the shell.

The only way I can see to make this reproducible is to archive every version of `named.root` that is published on our own. It doesn't seem highly worthwhile to do that though.